tests/test.py

```diff
@@ -34,9 +34,7 @@
 protocol = get_protocol("MyDatabase.Protocol.MyProtocol")
 assert isinstance(protocol, Protocol)
 
-speaker_diarization = get_protocol(
-    "MyDatabase.SpeakerDiarization.MySpeakerDiarization"
-)
+speaker_diarization = get_protocol("MyDatabase.SpeakerDiarization.MySpeakerDiarization")
```

Comment:
Looks like change in `tests/test.py` are unrelated to this PR main topic.
Can you revert those changes?